app/jobs/onceoff/fix_primary_emails_for_staged_users.rb

```diff
@@ -2,18 +2,25 @@ module Jobs
   class FixPrimaryEmailsForStagedUsers < Jobs::Onceoff
     def execute_onceoff(args)
       users = User.where(active: false, staged: true).joins(:email_tokens)
-      destroyer = UserDestroyer.new(Discourse.system_user)
+      acting_user = Discourse.system_user
+      destroyer = UserDestroyer.new(acting_user)
 
       users.group("email_tokens.email")
         .having("COUNT(email_tokens.email) > 1")
         .count
         .each_key do |email|
 
-        users.where("email_tokens.email = ?", email)
-          .order(id: :asc)
-          .offset(1)
+        query = users.where("email_tokens.email = ?", email)
+                  .order(id: :asc)
```

Comment:
Indentation looks abit off here :grin: If it fits within 80 chars, we can just put it on the same line